sematic/BUILD

```diff
@@ -80,6 +80,7 @@ sematic_py_lib(
         "//sematic/db/models:run",
```

Comment:
Please add screenshots to the PR description with how these new situations you've listed are rendered in the logs tab `ExternalException` element.

Reply:
Added screenshots where the behavior in the logs tab differs from `main`, and added notes where it doesn't.